src/ios/CDVConnection.m

```diff
@@ -60,28 +60,39 @@ - (NSString*)w3cConnectionTypeFor:(CDVReachability*)reachability
+                    } else if ([currentRadioAccessTechnology  isEqualToString:CTRadioAccessTechnologyEdge]) {
                         return @"2g";
-                    } else if ([telephonyInfo.currentRadioAccessTechnology  isEqualToString:CTRadioAccessTechnologyWCDMA]) {
+                    } else if ([currentRadioAccessTechnology  isEqualToString:CTRadioAccessTechnologyWCDMA]) {
                         return @"3g";
-                    } else if ([telephonyInfo.currentRadioAccessTechnology  isEqualToString:CTRadioAccessTechnologyHSDPA]) {
+                    } else if ([currentRadioAccessTechnology  isEqualToString:CTRadioAccessTechnologyHSDPA]) {
                         return @"3g";
-                    } else if ([telephonyInfo.currentRadioAccessTechnology  isEqualToString:CTRadioAccessTechnologyHSUPA]) {
+                    } else if ([currentRadioAccessTechnology  isEqualToString:CTRadioAccessTechnologyHSUPA]) {
                         return @"3g";
-                    } else if ([telephonyInfo.currentRadioAccessTechnology  isEqualToString:CTRadioAccessTechnologyCDMA1x]) {
+                    } else if ([currentRadioAccessTechnology  isEqualToString:CTRadioAccessTechnologyCDMA1x]) {
                         return @"3g";
-                    } else if ([telephonyInfo.currentRadioAccessTechnology  isEqualToString:CTRadioAccessTechnologyCDMAEVDORev0]) {
+                    } else if ([currentRadioAccessTechnology  isEqualToString:CTRadioAccessTechnologyCDMAEVDORev0]) {
                         return @"3g";
-                    } else if ([telephonyInfo.currentRadioAccessTechnology  isEqualToString:CTRadioAccessTechnologyCDMAEVDORevA]) {
+                    } else if ([currentRadioAccessTechnology  isEqualToString:CTRadioAccessTechnologyCDMAEVDORevA]) {
                         return @"3g";
-                    } else if ([telephonyInfo.currentRadioAccessTechnology  isEqualToString:CTRadioAccessTechnologyCDMAEVDORevB]) {
+                    } else if ([currentRadioAccessTechnology  isEqualToString:CTRadioAccessTechnologyCDMAEVDORevB]) {
                         return @"3g";
-                    } else if ([telephonyInfo.currentRadioAccessTechnology  isEqualToString:CTRadioAccessTechnologyeHRPD]) {
+                    } else if ([currentRadioAccessTechnology  isEqualToString:CTRadioAccessTechnologyeHRPD]) {
                         return @"3g";
-                    } else if ([telephonyInfo.currentRadioAccessTechnology  isEqualToString:CTRadioAccessTechnologyLTE]) {
+                    } else if ([currentRadioAccessTechnology  isEqualToString:CTRadioAccessTechnologyLTE]) {
                         return @"4g";
+                    } 
+                    #if __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_14_1 
+                    else if (@available(iOS 14.1, *)) {
+                        if ([currentRadioAccessTechnology  isEqualToString:CTRadioAccessTechnologyNRNSA]) {
```

Comment:
the preprocessor macro doesn't appear to be working as intended here cause iOS 13 tests is failing on this line due to `CTRadioAccessTechnologyNRNSA` not being available.

```
/var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/tmp-3401-5V69QdTKGZQj/platforms/ios/HelloCordova/Plugins/cordova-plugin-network-information/CDVConnection.m:89:76: error: use of undeclared identifier 'CTRadioAccessTechnologyNRNSA'
                        if ([currentRadioAccessTechnology  isEqualToString:CTRadioAccessTechnologyNRNSA]) {
                                                                           ^
```

Reply:
this error not make much sense for me, I mean is in a block of code that if the version is iOS14 will try to access.

Also I read this error is related with the xcode 

https://github.com/dcloudio/native-docs/issues/30 
https://stackoverflow.com/questions/71585983/use-of-undeclared-identifier-ctradioaccesstechnologynr

Any suggestions? I have the last version of xcode and I will try to run it in iOS 13 simulator and check if I can reproduce